tests/Earthfile

```diff
@@ -1474,15 +1474,20 @@ test-shared-cache:
       --extra_args=" --secret=content=filecontents " \
       --output_contains="filecontents"
 
-test-with-docker-parallel:
-  LOCALLY
-  RUN cd with-docker-parallel; \
-    f=/tmp/earthly.output-$(cat /proc/sys/kernel/random/uuid); \
-    earthly -P +parallel 2>&1 | tee -a $f; \
-    order=$(cat $f | grep -Eo "### [[:digit:]]+"  | cut -d' ' -f2- | tr '\n' ' ' | tr -d '[:space:]'); \
-    rm $f; \
-    echo order:$order; \
-    (echo $order | grep -Eq ^1+2+3+4+5+$) && echo "no parallelization" && exit 1 || echo "parallelization";
+test-new-visitor:
```

Comment:
nit: names like "new" can stick around much longer than expected in codebases, can we rename this to something that'll stand the test of time (otherwise we'll end up with new-final-v2)

Reply:
whoops, looks like we already have "new" under `newVisitedCollection`, guess my comment isn't as relevant.

Reply:
I will leave the renaming for the target PR, since test name is related to implementation name